content/dev/features/dataset-folders.md

```diff
@@ -1,48 +1,45 @@
 
-Dataset folders allow datasets to be organized into folders similar to what you would see in a file system like Windows Explorer or MacOS Finder. Folders make finding, organizing, and sharing data more intuitive. Here are some key differences from the old interface.
+* Folders can be used to organize large collections of datasets into more manageable units. If your organization imports data from many sources or has a large archive of previously imported data, you can organize them by source, time of import/survey, subject, etc.
 
-### Navigating
-#### 1. Project icons have been replaced by folders
+* Editors can grant access to different folders to different sets of users. Multiple clients, or multiple groups or teams within a client, can be granted access to just the data you want to show them. Furthermore, you can move a dataset into a folder to instantly make it available to all users who have access to that folder.
 
-{{< figure src="dev/features/images/folders-before-and-after.png" class="img-fluid">}}
+These new features in turn bring some changes to our web application. Here are some key differences from the old interface.
 
-Folders can be sorted by clicking the column headers at the top of the list.
+## Navigating
 
-Note that while we have removed the custom project icons from prominence in the interface for now, we will be reintroducing them along with additional branding settings next year.
+The first thing you'll notice is that the stripe of icons at the left have been replaced by folders. The flat list of project icons didn't work well with a model where folders could nest within each other, so we changed the display to more closely match the underlying model. (Note that while we have removed the custom icons from display for now, we will be reintroducing them with additional branding settings in the near future.)
 
-#### 2. The new Shared With Me folder
-Previously, any datasets that were shared with you, as well as any datasets that you had imported yourself, were stored together in the Personal Project project. Now you will see two folders, **Personal** and **Shared with Me**.
+{{< figure src="dev/features/images/folders-before-and-after.png" class="img-fluid">}}
 
-**Personal** will only appear to users who have permission to import data. It contains any datasets you have imported and not moved to another folder.
+Click on a folder's name to enter it. When you do, you'll see at the top of the screen the folder you are currently in and the path to that folder.
 
-**Shared with Me** contains datasets that have been shared with you directly without access being granted to the folder that contains it. If no datasets have been shared with you, this folder will be hidden.
+{{< figure src="dev/features/images/folders-breadcrumb.png" class="img-fluid">}}
```

Comment:
It would be nice if this screenshot matched the above screenshot (i.e. the folder name in the breadcrumbs should match a folder in the example)